pyuploadcare/api.py

```diff
@@ -65,6 +65,11 @@ def _build_user_agent():
                                                       extension_info)
 
 
+def _signed_upload_signature(secret, expire):
+    k, m = str(secret).encode(), str(expire).encode()
```

Comment:
What does `str(secret)` mean? This is not a proper way to convert bytes to string:

```python
>>> str(b'0123456789abcdef')
<<< "b'0123456789abcdef'"
```

Reply:
we don't need bytes to string, we need vice versa for hmac.
this was caught by tests on py3 @andrewshkovskii

Reply:
> we don't need bytes to string

What does `str(secret)` mean?

Reply:
we want to cast user's input to `string` (e.g. `None` or numbers) and `expire` is always a number

Reply:
Ok, this makes sence for `expire`, but leads to difficult to find errors if the `secret` is already bytes.